### PR TITLE
Catch exception per entity and log entity on failure.

### PR DESCRIPTION
**Pull request by Jan-M**
Catch exception per entity and log entity on failure.

### CODE REVIEW

**Comment by mohabusama** [comment]
👍

File: zmon_agent/main.py [hunk 1]
```diff
@@ -92,19 +92,17 @@ def add_new_entities(all_current_entities, existing_entities, zmon_client, dry_r
...
                 resp.raise_for_status()
-        except Exception:
-            current_span.set_tag('error', True)
-            logger.exception('Failed to add entity!')
-            current_span.log_kv({'exception': traceback.format_exc()})
-            error_count += 1
+            except Exception:
+                current_span.set_tag('error', True)
+                logger.exception('Failed to add entity!')
+                current_span.log_kv({'exception': traceback.format_exc(), "entity": entity})
```
Comment (mohabusama) [inline zmon_agent/main.py:104]: may be just ``'entity_id': entity['id']``

**Comment by Jan-M** [comment]
:+1:

File: zmon_agent/main.py (hunk 1 above)
Comment (Jan-M) [reply zmon_agent/main.py:104]: hmm, i kind of did it on purpose, want to see the rendered entity in case this is a problem for the 500s

Comment (mohabusama) [reply zmon_agent/main.py:104]: Alright

**Pull request merged**